src/SelfMeasurements-2-0/Test/PHR-Client/selfmeasurements-phr-send-01-01-respiration.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="http://hl7.org/fhir/STU3/testscript.sch" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TestScript xmlns="http://hl7.org/fhir" xmlns:nts="http://nictiz.nl/xsl/testscript"
+    nts:scenario="server">
+    <id value="phr-send-01-01-respiration"/>
+    <version value="stu3-2.0"/>
+    <name
+        value="SelfMeasurements - PHR Server - Scenario 1.1 - Send most recent respiration Observation resource"/>
+    <description value="Scenario 1.1 - Send most recent respiration Observation resource"/>
+    <nts:patientTokenFixture href="selfmeasurements-XXX-Herder-token.xml"/>
+    <nts:fixture id="scenario-1-1-bundle-fixture"
+        href="bundles/send/selfmeasurements-send-scenario-1-1-respiration-bundle.xml"/>
+    <nts:includeDateT value="yes"/>
+
+    <test id="scenario1-1-receive-most-recent-respiration">
+        <name value="Scenario 1.1"/>
+        <description value="Send most recent respiration Observation resource"/>
+        <nts:include value="medmij/test.phr.successfulBatch" scope="common"
+            sourceId="scenario-1-1-bundle-fixture"/>
+        <nts:include value="assert.response.numResources" scope="common"
+            resource="Observation" count="1"/>
+        <nts:include value="assert.response.numResources" scope="common"
+            resource="Patient" count="1"/>
```

Comment:
Ik twijfel of het meesturen van de Patient verplicht is. Zie het niet op de TO terugkomen.